src/pages/ReportParticipantsPage.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import _ from 'underscore';
+import {
+    View,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import styles from '../styles/styles';
+import ONYXKEYS from '../ONYXKEYS';
+import HeaderWithCloseButton from '../components/HeaderWithCloseButton';
+import Navigation from '../libs/Navigation/Navigation';
+import ScreenWrapper from '../components/ScreenWrapper';
+import OptionsList from '../components/OptionsList';
+import ROUTES from '../ROUTES';
+
+const propTypes = {
+    /* Onyx Props */
+    // The personal details of the person who is logged in
+    personalDetails: PropTypes.shape({
+        // Display name of the current user from their personal details
+        displayName: PropTypes.string,
+
+        // Avatar URL of the current user from their personal details
+        avatar: PropTypes.string,
+
+        // Login of the current user from their personal details
+        login: PropTypes.string,
+    }).isRequired,
+
+    // The active report
+    report: PropTypes.shape({
+        // The list of icons
+        icons: PropTypes.arrayOf(PropTypes.string),
+
+        // The report name
+        reportName: PropTypes.string,
+
+        // Array of participants
+        participants: PropTypes.arrayOf(PropTypes.string),
+    }).isRequired,
+
+    // Route params
+    route: PropTypes.shape({
+        params: PropTypes.shape({
+            // Report ID passed via route r/:reportID/participants
+            reportID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+/**
+ * Returns all the participants in the active report
+ *
+ * @param {Object} report The active report object
+ * @param {Array} personalDetails The personal details of the users
+ * @return {Array}
+ */
+const getAllParticipants = (report, personalDetails) => {
+    const {participants} = report;
+
+    return _.map(participants, (login) => {
+        const userPersonalDetail = personalDetails[login];
+
+        return ({
+            alternateText: login,
+            displayName: userPersonalDetail.displayName,
+            icons: [userPersonalDetail.avatar],
+            keyForList: login,
+            login,
```

Comment:
We should use `Str.removeSMSDomain(login)` wherever we use `login` to ensure we're not displaying `@expensify.sms` for SMS logins. 

![image](https://user-images.githubusercontent.com/16747822/114121920-22276e00-98a4-11eb-9df6-6cd4e0cb5f2f.png)